new-starter-guide/new-starter-to-origami.md

```diff
@@ -116,11 +118,11 @@ You can join all the Origami applications on Heroku by checking out this [shared
 
 #### Issues with installing/getting access
 
-If you experience any problems, then please take a look at Next's [troubleshooting guide](http://financial-times.github.io/next/docs/developer-guide/troubleshooting/). If your problem is not included in the troubleshooting list and you manage to solve it, then please add it to the list.
+If you experience any problems, then please take a look at Next's troubleshooting guide. If your problem is not included in the [troubleshooting guide](http://financial-times.github.io/next/docs/developer-guide/troubleshooting/) and you manage to solve it, then add it to the list.
 
 ### Peer review and pull requests
 
-All code in Origami is written by one person and peer reviewed by one or more people using git branches and merging.
+One person writes all the code in a Origami module. The code will be peer reviewed by one or more people using git branches and merging.
```

Comment:
It's not always an Origami module, this has to apply to services and other projects too. Suggest changing to something like:

> All contributors to Origami must use Pull Requests. These should be peer-reviewed by one or more people before merging.